test-workflow.yml

```diff
@@ -20,9 +24,9 @@ jobs:
 
     - name: Configure AWS credentials
       uses: aws-actions/configure-aws-credentials@v4
-      with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      with:        
```

Comment:
nit: please remove the extra spacing at the end of this line.

Reply:
Addressed the feedback.